src/app/zap-templates/zcl/data-model/chip/electrical-energy-measurement-cluster.xml

```diff
@@ -0,0 +1,108 @@
+    <item fieldId="1" name="RangeMax" type="int64s" />
+    <item fieldId="2" name="PercentMax" type="percent100ths" optional="true" />
+    <item fieldId="3" name="PercentMin" type="percent100ths"  optional="true"/>
+    <item fieldId="4" name="PercentTypical" type="percent100ths"  optional="true"/>
+    <item fieldId="5" name="FixedMax" type="int64u"  optional="true"/>
+    <item fieldId="6" name="FixedMin" type="int64u"  optional="true"/>
+    <item fieldId="7" name="FixedTypical" type="int64u"  optional="true"/>
+  </struct>
+
+  <struct name="MeasurementAccuracyStruct">
+    <cluster code="0x0091"/>
+    <item fieldId="0" name="MeasurementType" type="MeasurementTypeEnum" />
+    <item fieldId="1" name="Measured" type="boolean" />
+    <item fieldId="2" name="MinMeasuredValue" type="int64s" />
+    <item fieldId="3" name="MaxMeasuredValue" type="int64s" />
+    <item fieldId="4" name="AccuracyRanges" type="MeasurementAccuracyRangeStruct" array="true" />
+  </struct>
+
+  <cluster apiMaturity="provisional">
+    <name>Electrical Energy Measurement</name>
+    <domain>Measurement &amp; Sensing</domain>
+    <description>This cluster provides a mechanism for querying data about the electrical energy imported or provided by the server.</description>
+    <code>0x0091</code>
+    <define>ELECTRICAL_ENERGY_MEASUREMENT_CLUSTER</define>
+    <client tick="false" init="false">true</client>
+    <server tick="false" init="false">true</server>
+    <globalAttribute side="either" code="0xFFFD" value="1"/> <!-- Revision   -->
+
+    <attribute side="server" code="0x0000" define="MEASURED" type="boolean" writable="false" default="false" optional="false">Measured</attribute>
+    <attribute side="server" code="0x0001" define="ACCURACY" type="MeasurementAccuracyStruct" writable="false" default="false" optional="false">Accuracy</attribute>
+    <attribute side="server" code="0x0002" define="CUMULATIVE_ENERGY_IMPORTED" type="EnergyMeasurementStruct" writable="false" default="false" optional="true" isNullable="true">CumulativeEnergyImported</attribute>
+    <attribute side="server" code="0x0003" define="CUMULATIVE_ENERGY_EXPORTED" type="EnergyMeasurementStruct" writable="false" default="false" optional="true" isNullable="true">CumulativeEnergyExported</attribute>
+    <attribute side="server" code="0x0004" define="PERIODIC_ENERGY_IMPORTED" type="EnergyMeasurementStruct" writable="false" default="false" optional="true" isNullable="true">PeriodicEnergyImported</attribute>
+    <attribute side="server" code="0x0005" define="PERIODIC_ENERGY_EXPORTED" type="EnergyMeasurementStruct" writable="false" default="false" optional="true" isNullable="true">PeriodicEnergyExported</attribute>
+
+    <event side="server" code="0x00" name="CumulativeEnergyMeasured" priority="info" optional="true">
+      <description>This event SHALL be generated when the server takes a snapshot of th cumulative energy imported by the server.</description>
```

Comment:
```suggestion
      <description>This event SHALL be generated when the server takes a snapshot of the cumulative energy imported by the server, exported from the server, or both.</description>
```

Reply:
Think this change is still missing.

Reply:
Not sure how I missed it... but updated now (I hope).